buildroot-external/patches/occu/0207-WebUI-ModernLoginPage/occu/WebUI/www/rega/login.css

```
@@ -0,0 +1,326 @@
+html, body {
+  height: 100%;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif;
+  background: #0f0f11;
+  color: #f0f0f0;
+}
+
+/* ── Layout ─────────────────────────────────────────────── */
+#maintable {
+  display: flex;
+  flex-direction: row;
+  height: 100vh;
```

Stylelint (17.6.0)

[error] Unexpected duplicate "height" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)